### PR TITLE
fix: account linking params should be inverted

### PR DESCRIPTION
**Pull request by danielgent**
I recreated https://github.com/sesamyab/auth/pull/367

### CODE REVIEW

File: src/routes/management-api/users.ts [hunk 1]
```diff
@@ -265,6 +265,7 @@ export class UsersMgmtController extends Controller {
     return identities;
   }
 
+  // what should happen here?  Where we do we specify the linked user? what should this even do?
```
Comment (danielgent) [inline src/routes/management-api/users.ts:268]: :eyes: What should happen here? we have tests checking this. TBD anyway

**Comment by danielgent** [comment]
@markusahlstrand I'll merge this as it's essentially what was on your PR.

File: src/routes/management-api/users.ts (hunk 1 above)
Comment (markusahlstrand) [reply src/routes/management-api/users.ts:268]: Guess it just removes the pointer right? Do you think it would be using the wrong id?

Comment (danielgent) [reply src/routes/management-api/users.ts:268]: Does it work on HQ? Do we have this functionality on HQ @markusahlstrand ?

Now when linking `@Path("user_id") userId: string,` is the user we are linking to (the primary account)

So now the unlink method is unlinking the primary account? or does `user_id` change here and become the secondary linked account?

If you're QAing all this though in HQ then no need to discuss :+1:

**Pull request merged**